core/model/legacyEvents.go

```diff
@@ -93,6 +93,13 @@ type WebToolsEvent struct {
 	} `xml:"topic"`
 }
 
+// WebToolsEventID represents web tools blacklist ids
+type WebToolsEventID struct {
```

Comment:
We have discussed the format of the data to be:

The first item in the collection:
```
{
  “_id”: "1", 
  “name”: “webtools_events_ids”,
  “data”: [
    “55555”,
    “44444"
  ]
}
```
And the second item to be:
```
{
  “_id”: "2", 
  “name”: “webtools_calendar_ids”,
  “data”: [
    “77777”
  ]
}
```

Instead of this:
```
{
  “_id”: {
    “$oid”: “65fd4b961934b6f10dc4cd3b”
  },
  “name”: “webtools_events_ids”,
  “data_source_ids”: [
    “55555”,
    “44444"
  ],
  “data_calendar_ids”: [
    “77777"
  ]
}
```